src/data/grammar/mtg-keep-or-mulligan.yaml

```diff
@@ -86,12 +86,6 @@ svgMulliganBeforeTextStyle: >
 svgMulliganAfterTextStyle: >
   color: rgb(254,255,250); font-size: 32px; font-family: Sans-Serif; font-weight: 600;
   text-shadow: 1px 1px 1px rgb(60,60,60); text-align:center;
-svgMulliganCardId:
```

Comment:
I actually like using these land references, since it makes it more likely to generate reasonable-looking hands (rather than card salad, which uh, "breaks immersion", I guess). We can update that later, though.